minitests/roi_harness/runme.sh

```diff
@@ -9,6 +9,16 @@ source ${XRAY_DIR}/utils/environment.sh
 export XRAY_PINCFG=${XRAY_PINCFG:-ARTY-A7-SWBUT}
 export BUILD_DIR=${BUILD_DIR:-build}
 
+DEVICE=`expr substr $XRAY_PART 4 1`
```

Comment:
This should be checking if $XRAY_DATABASE is zynq7

Reply:
Actually looks like its using the definition of XRAY_DATABASE. Please submit a follow up PR to remove this in leiu of XRAY_DATABASE

Reply:
done in #433